packages/core/test-project/ios/RNEmbraceCoreTests/RNEmbraceCoreTests.swift

```diff
@@ -731,11 +735,13 @@ class EmbraceManagerTests: XCTestCase {
         XCTAssertEqual(exportedSpans[0].attributes["http.response.body.size"]!.description, "2000")
         XCTAssertEqual(exportedSpans[0].attributes["http.request.body.size"]!.description, "1000")
         XCTAssertEqual(exportedSpans[0].attributes["http.response.status_code"]!.description, "200")
+        XCTAssertNotNil(exportedSpans[0].attributes["emb.w3c_traceparent"])
```

Comment:
is there a way to test the case where the config is off and we can verify the attribute isn't set? Might not be possible if that involves starting the SDK from scratch again since we want to avoid that in testing

Reply:
confirmed by Ari, there is no way of updating the config through the code (it's inmutable). not sure if there is other way of mocking the `config` variable and make it return a falsy value for the `disableNetworkSpanForwarding` key

Reply:
I wouldn't spend too much time on it, it's something we can punt more to an integration test and spin up an app with a different config

for the purposes of this testing I guess could maybe move the `if` check inside the method and pass in the config as a parameter, could then have a unit test just on that method and confirm the attribute being added or not...though that may not add too much value